### ISSUE TITLE
Typing < adds a >

### ISSUE THREAD

**Issue by dospunk** [zargony/atom-language-rust#122]
`<` is also used for comparison, so auto-completing it to `<>`  feels clunky

**Comment by zargony** [comment]
You're right. For generic types it makes sense to autocomplete `<>`, for comparison, it doesn't. Unfortunately, the package can't distinguish between these cases as far as I know.

**Comment by sunjay** [comment]
Would it be possible to add a way to just disable the autocomplete then? I find deleting the extra `>` more annoying than having to type it.
